Seq.Input.CertificateCheck/Extensions/Certificate2Extensions.cs

```diff
@@ -14,9 +14,8 @@ public static IEnumerable<string> ParseSubjectAlternativeNames(this X509Certific
             Regex sanRex = new Regex(@"^DNS Name=(.*)", RegexOptions.Compiled | RegexOptions.CultureInvariant);
 
             var sanList = from X509Extension ext in cert.Extensions
-                where ext.Oid.FriendlyName.Equals("Subject Alternative Name", StringComparison.Ordinal)
-                let data = new AsnEncodedData(ext.Oid, ext.RawData)
-                let text = data.Format(true)
+                where "2.5.29.17".Equals(ext.Oid.Value, StringComparison.Ordinal) // OID for Subject Alternative Name
```

Comment:
@TwelveBaud Just looking at this now, apologies for the late responses, it doesn't seem right to check if 2.5.29.17 is the Old?

Reply:
NVM :joy: